examples/notebooks/Date Picker Widget.ipynb

```diff
@@ -817,21 +807,21 @@
    ]
   ],
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```

Comment:
Fixed. There were a couple of notebooks that were already set to Python 2, which I reverted to python. All squashed in earlier commits.